### PR TITLE
RM-734 Update versions for 1.5.0 release

### PR DESCRIPTION
**Pull request by tplavcic**
[RM-734](https://jira.percona.com/browse/RM-734)

deploy/bundle.yaml changes not included right now since not sure what is needed

### CODE REVIEW

File: deploy/operator.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ spec:
       serviceAccountName: percona-xtradb-cluster-operator
```
Comment (delgod) [inline deploy/operator.yaml:15]: please regenerate bundle.yaml also

File: version/version.go [hunk 2]
```diff
@@ -1,5 +1,5 @@
 package version
 
 var (
-	Version = "1.5.0"
+	Version = "1.6.0"
```
Comment (delgod) [inline version/version.go:4]: hm, we will create release branch before or after this commit?
part of changes needed for release branch, but this line is not needed for release branch

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| recreate | passed |
| big-data | failed |
| storage | failed |
| scaling | passed |
| self-healing | passed |
| scaling-proxysql | passed |
| limits | passed |
| upgrade | passed |
| restore-to-encrypted-cluster | passed |
| monitoring | passed |
| self-healing-advanced | failed |
| upgrade-consistency | passed |
| monitoring-2-0 | passed |
| security-context | passed |
| demand-backup | failed |
| affinity | passed |
| operator-self-healing | failed |
| one-pod | failed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| scheduled-backup | failed |
| demand-backup-encrypted-with-tls | failed |

File: version/version.go (hunk 2 above)
Comment (tplavcic) [reply version/version.go:4]: The intention was to merge this and then to create release branch. I'll remove this, but what about change in `deploy/crd.yaml` - should it be removed also?

File: deploy/operator.yaml (hunk 1 above)
Comment (tplavcic) [reply deploy/operator.yaml:15]: @cap1984 Could you please share the steps to regenerate bundle.yaml? Thanks!

**Comment by tplavcic** [comment]
@delgod @cap1984 Made fixes, please re-check.

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| scaling | passed |
| scaling-proxysql | passed |
| upgrade | passed |
| upgrade-consistency | passed |
| security-context | passed |
| recreate | passed |
| restore-to-encrypted-cluster | passed |
| demand-backup | passed |
| storage | passed |
| init-deploy | passed |
| limits | passed |
| monitoring | passed |
| monitoring-2-0 | passed |
| big-data | failed |
| affinity | passed |
| one-pod | failed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| self-healing | passed |
| self-healing-advanced | failed |
| operator-self-healing | passed |
| scheduled-backup | failed |
| demand-backup-encrypted-with-tls | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | passed |
| init-deploy | passed |
| scaling | passed |
| big-data | failed |
| recreate | passed |
| self-healing | failed |
| limits | passed |
| monitoring | passed |
| restore-to-encrypted-cluster | failed |
| scaling-proxysql | passed |
| demand-backup | passed |
| self-healing-advanced | failed |
| upgrade | passed |
| monitoring-2-0 | passed |
| upgrade-consistency | passed |
| operator-self-healing | passed |
| security-context | passed |
| affinity | passed |
| scheduled-backup | failed |
| one-pod | passed |
| auto-tuning | passed |
| demand-backup-encrypted-with-tls | failed |
| proxysql-sidecar-res-limits | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | failed |
| scaling | failed |
| storage | failed |
| big-data | failed |
| recreate | failed |
| self-healing | failed |
| scaling-proxysql | failed |
| restore-to-encrypted-cluster | failed |
| self-healing-advanced | failed |
| upgrade | failed |
| operator-self-healing | failed |
| demand-backup | failed |
| scheduled-backup | failed |
| limits | failed |
| demand-backup-encrypted-with-tls | failed |
| monitoring | failed |
| monitoring-2-0 | failed |
| affinity | failed |
| upgrade-consistency | failed |
| one-pod | failed |
| auto-tuning | failed |
| proxysql-sidecar-res-limits | failed |
| security-context | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | passed |
| scaling | passed |
| init-deploy | passed |
| big-data | failed |
| recreate | passed |
| self-healing | passed |
| scaling-proxysql | passed |
| limits | passed |
| monitoring | passed |
| restore-to-encrypted-cluster | passed |
| upgrade | passed |
| monitoring-2-0 | passed |
| upgrade-consistency | passed |
| self-healing-advanced | failed |
| security-context | passed |
| demand-backup | failed |
| operator-self-healing | passed |
| affinity | passed |
| scheduled-backup | failed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| demand-backup-encrypted-with-tls | passed |

**Pull request merged**